RZAssemblage/RZAssemblage.h

```diff
@@ -1,19 +1,127 @@
 //
-//  RZAssemblage.h
+//  RZBaseAssemblage.h
 //  RZAssemblage
 //
 //  Created by Brian King on 1/27/15.
 //  Copyright (c) 2015 Raizlabs. All rights reserved.
 //
 
-#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
 
-//! Project version number for RZAssemblage.
 FOUNDATION_EXPORT double RZAssemblageVersionNumber;
-
-//! Project version string for RZAssemblage.
 FOUNDATION_EXPORT const unsigned char RZAssemblageVersionString[];
```

Comment:
Here's the output for those as of right now:
```
(lldb) p RZAssemblageVersionNumber
(const double) $0 = 1
(lldb) p RZAssemblageVersionString
(const unsigned char [50]) $1 = "@(#)PROGRAM:RZAssemblage  PROJECT:RZAssemblage-1\n"
```
I think they're pulled from the `Version` field of the Xcode project? I'm not sure, but by default they look pretty useless.